projects/sel4/porting.md

```diff
@@ -7,79 +7,159 @@ SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 
 # Porting seL4 to a new platform
 
+This section covers porting seL4 to a new ARM or RISC-V platform. You can check the list of supported seL4 platforms [here](https://docs.sel4.systems/Hardware/).
+
 ## Setup
 
 In order to load the seL4 image (via some means) it is important to have a working bootloader.
-We tend to use [U-boot](https://www.denx.de/wiki/U-Boot) as it is open source, reliable and supports many different architectures.
+We tend to use [U-Boot](https://www.denx.de/wiki/U-Boot) as it is open source, reliable and supports many different architectures.
+
+There are a variety of ways of loading an seL4 image from U-Boot such as eMMC, TFTP, and USB.
 
-For initial setup you can follow the instructions on the UNSW advanced operating systems website regarding [setting up a tftp server](https://www.cse.unsw.edu.au/~cs9242/19/project/linux.shtml).
+For setting up a TFTP server to transfer images over the network, you can follow the
+[UNSW Advanced Operating Systems course guide](https://www.cse.unsw.edu.au/~cs9242/19/project/linux.shtml).
 
-Once you get U-boot up and running it is a good idea (if the platform is supported) to flash a
-Linux image onto the dev board. This way you can find the correct address to load the image into memory.
+Once you get U-Boot up and running it is a good idea (if the platform is supported) to flash a
+Linux image onto the board. This way you can find the correct address to load the image into memory.
 This is also useful for debugging, as you can use the FDT file system to dump addresses from a running system.
 
 Once you have found the correct address to load, you can use the command `bootcmd="tftpboot 0x20000000 sel4_image && go 0x20000000"`
-(using 0x20000000 as an example) followed by `saveenv` to save the starting address onto the boards flash memory.
-You will also need to set the 'ipaddr' and 'serverip' details can be found in the link above.
+(using `0x20000000` as an example) followed by the `saveenv` command to save the starting address onto the boards flash memory.
```

Comment:
Maybe make this a command block also:

Once you have found the correct address to load, declare the boot command (where address `0x20000000` as an example) 
```
bootcmd="tftpboot 0x20000000 sel4_image && go 0x20000000"
```
You will also need to set the `ipaddr` and `serverip` environment variables, details can be found in the link above. Finally, Save the U-Boot settings in the boards flash memory.
```
saveenv
```